packages/theme/src/cli/utilities/theme-command.ts

```diff
@@ -4,15 +4,15 @@ interface FlagValues {
   [key: string]: boolean | string | string[] | number | undefined
 }
 interface PassThroughFlagsOptions {
-  // Exclude flags that are only for CLI3 but will cause errors if passed to CLI2
-  exclude?: string[]
+  // Only pass on flags that are relevant to CLI2
+  relevantFlags?: string[]
 }
 
 export default abstract class ThemeCommand extends Command {
-  passThroughFlags(flags: FlagValues, {exclude}: PassThroughFlagsOptions): string[] {
+  passThroughFlags(flags: FlagValues, {relevantFlags}: PassThroughFlagsOptions): string[] {
```

Comment:
What about adding the default value here?

```suggestion
  passThroughFlags(flags: FlagValues, {relevantFlags: Array = []}: PassThroughFlagsOptions): string[] {
```

Reply:
This doesn't work as cleanly with TypeScript, I believe we'd have to do:

```suggestion
  passThroughFlags(flags: FlagValues, {relevantFlags}: PassThroughFlagsOptions = {relevantFlags: []}): string[] {
```

Also note it's a default value for the whole object, so if `{}` is passed then `relevantFlags` would still be `undefined`.